CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### Grafana Mimir
 
+* [BUGFIX] Fix sanity check done on configured filesystem directories when running Alertmanager in microservices mode. #2947
```

Comment:
Changelog on the release branch should not mention `main / unreleased`.

Reply:
But it does. I guess we'll fix it before publishing the stable version.

Reply:
> But it does.

Yep. 😂 (It has nothing to do with this PR, I should have been more careful reviewing the other PR that introduced the change.)